test/functional/net.py

```diff
@@ -32,6 +32,42 @@ def setup_network(self):
         self.sync_all()
 
     def run_test(self):
+        self._test_connection_count()
+        self._test_getnettotals()
+        self._test_getnetworkinginfo()
+        self._test_getaddednodeinfo()
+
+    def _test_connection_count(self):
+        assert_equal(self.nodes[0].getconnectioncount(), self.num_nodes)
```

Comment:
nit: I wouldn't assert these two things are equal. Number of nodes is 2, and the test framework sets up two connections between each node (see `connect_nodes_bi()`). The fact that these are the same number is a coincidence.

Just `assert_equal(self.nodes[0].getconnectioncount(), 2)` is better, with a comment explaining why.